vocabularies/gsq-roles.ttl

```diff
@@ -0,0 +1,80 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix gsqr: <http://linked.data.gov.au/def/gsq-roles/> .
+@prefix iso11179: <http://linked.data.gov.au/def/iso11179-6/RolesAndResponsibilities/> .
+@prefix isorole: <http://linked.data.gov.au/def/iso19115-1/RoleCode/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<http://linked.data.gov.au/def/gsq-roles> a owl:Ontology .
+
+gsqr:conceptScheme a skos:ConceptScheme ;
+    dct:source <https://standards.iso.org/iso/19115/resources/Codelists/gml/CI_RoleCode.xml> ;
+    skos:definition "The roles of Agents (People and Organisations) in relation to Entities (Datasets, Samples etc.)"@en ;
+    skos:hasTopConcept gsqr:contractor,
+        iso11179:Submitter,
+        isorole:author,
+        isorole:owner,
+        isorole:publisher ;
+    skos:prefLabel "Geological Survey of Queensland's Agent Roles"@en .
+
+gsqr:QdexReportsRoles a skos:Collection ;
+    skos:definition "A collection of roles used for Agents within the QDEX Reports regime"@en ;
+    skos:member gsqr:contractor,
+        gsqr:driller,
+        gsqr:operator,
+        iso11179:Submitter,
+        isorole:author,
+        isorole:owner,
+        isorole:publisher ;
+    skos:prefLabel "Qdex Reports Roles"@en .
+
+gsqr:driller a skos:Concept ;
+    skos:broader gsqr:contractor ;
+    skos:definition "Driller"@en ;
+    skos:inScheme isorole:conceptScheme ;
+    skos:prefLabel "Contractors who drill bores/wellbores/boreholes etc."@en ;
+    skos:topConceptOf gsqr:conceptScheme .
```

Comment:
Not always necessarily a contractor. A driller can be the operating company who in the specific instance happens to be the driller of a well or bore.
"The agent who drills or bores the borehole"

Reply:
Addressed as recommended by commit https://github.com/geological-survey-of-queensland/vocabularies/pull/60/commits/59061ec98bd0c0d9baf04195ec0695d1bc4705e4